docs/components/campaigns.rst

```diff
@@ -855,3 +855,125 @@ The Campaign Engine then dispatches the Decision Event's ``eventName`` where lis
+
+The export command can be called from the command line or via the API as well as in the Mautic UI.
+
+
+Exporting via Command line
+==========================
+
+First, copy the campaign zip file into the correct place to be imported into Mautic.
+
+.. code-block:: bash
+
+   docker cp ./campaign_data.zip ddev-mautic-web:/tmp/entity_data.zip
+
+Then, run the import command:
+
+.. code-block:: bash
+
+   bin/console mautic:entity:import --entity=campaign --file=/tmp/entity_data.zip --user=*{<user_id>}*
+
+Options:
+
+=====================  ============================================================
+Option                 Description
+=====================  ============================================================
+``--entity=campaign``  Specifies the entity type being imported (e.g. campaign)
+``--file``             Path to the file you want to import (ZIP)
+``--user=``            User ID to associate with the import process (e.g. 1 (admin))
+=====================  ============================================================
+
+Exporting via API
+=================
+
+Endpoint:
+``GET https://{*your-mautic-domain*}/api/campaigns/export/{*<campaign_id>*}``
+
+The final part of the URL, ``{*<campaign_id>*}``, specifies the ID of the campaign to be exported.
+
+Headers:
+
+=================  ===========================
+Header             Description
+=================  ===========================
+``Authorization``  Bearer *<access_token>*
+=================  ===========================
+
+Example Request (cURL):
+
+.. code-block:: bash
+
+   curl --location 'https://{*your-mautic-domain*}/api/campaigns/export/{*<campaign_id>*}' \
+   --header 'Authorization: Bearer *<your_actual_access_token>*' \
+   --data ''
+
+:download:`Example campaign JSON file <images/campaign-sample.json>`
+
+
+Importing a campaign
+********************
+
+Mautic allows you to import campaigns using JSON files that were previously exported. This feature helps you transfer campaigns between Mautic instances.
+
+During the import process, Mautic performs the following actions:
+
+*   Verifies that the importing user has the necessary permissions and access to all relevant entities.
+*   Checks if all plugins and other dependencies required by the campaign are present in the current Mautic instance.
+*   Identifies any ID conflicts with existing entities. If conflicts are found, Mautic will prompt you to either update the existing entities or create new ones.
+*   Creates the campaign along with all its related entities to ensure it functions correctly after import.
+
+You can trigger the import command from the command line or via API.
+
+Importing via Command line
+==========================
+
+First, copy the campaign data ZIP file to your DDEV container:
+
+.. code-block:: bash
+
+   docker cp ./campaign_data.zip ddev-mautic-web:/tmp/entity_data.zip
```

Comment:
Same as above. We should mention to use other means to copy the file to their production. No need for examples, just something users can google for.